src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+    private static String SYMBOL = "E";
```

Comment:
Similar to previous comments about constants, you can consider making this a final variable its value should not be changing